### PR TITLE
Feature: Approval step for ERC20 tokens

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- Adds approval step for ERC20 Tokens
- updates SDK to latest version
- fixes gas estimation fee defaulting to error

Task: [APP-530](https://aragonassociation.atlassian.net/browse/APP-530)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/hooks/usePollGasfee.tsx [hunk 1]
```diff
@@ -22,9 +22,9 @@ export const usePollGasFee = (
   shouldPoll = true
 ) => {
   const {network} = useNetwork();
-  const [error, setError] = useState<Error | undefined>(undefined);
-  const [maxFee, setMaxFee] = useState<BigInt | undefined>(undefined);
-  const [averageFee, setAverageFee] = useState<BigInt | undefined>(undefined);
+  const [error, setError] = useState<Error | undefined>();
+  const [maxFee, setMaxFee] = useState<BigInt | undefined>(BigInt(0));
```
Comment (sepehr2github) [inline packages/web-app/src/hooks/usePollGasfee.tsx:26]: Screen Shot 2022-06-20 at 3 24 39 PM
These new changes caused a build issue, Can you take a look?

File: packages/web-app/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@apollo/client": "^3.5.8",
-    "@aragon/sdk-client": "0.0.13-alpha",
+    "@aragon/sdk-client": "^0.0.17-alpha",
```
Comment (sepehr2github) [inline packages/web-app/package.json]: You can now update to v0.0.18

**Review by sepehr2github — APPROVED** [review]
It's totally fine by my side. only one thing left which is a build error!

File: packages/web-app/src/hooks/usePollGasfee.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/hooks/usePollGasfee.tsx:26]: Are you sure you used the latest version of the branch? The last commit should have taken care of that..

Comment (sepehr2github) [reply packages/web-app/src/hooks/usePollGasfee.tsx:26]: No I'm not sure ;D hahaha

Comment (sepehr2github) [reply packages/web-app/src/hooks/usePollGasfee.tsx:26]: let me check again

Comment (sepehr2github) [reply packages/web-app/src/hooks/usePollGasfee.tsx:26]: You were right! good to go!

**Review by RakeshUP — APPROVED** [review]

**Pull request merged**